claat/render/md.go

```diff
@@ -232,3 +233,37 @@ func (mw *mdWriter) header(n *types.HeaderNode) {
 		mw.writeBytes(newLine)
 	}
 }
+
+func (mw *mdWriter) grid(n *types.GridNode) {
+	mw.newBlock()
+	mw.writeString(`<table markdown="1">`)
+	for _, r := range n.Rows {
+		mw.writeString("\n<tr>")
+		for _, c := range r {
+			mw.writeFmt(`<td colspan="%d" rowspan="%d">`, c.Colspan, c.Rowspan)
+			mw.write(c.Content.Nodes...)
+			mw.writeString("</td>")
+		}
+		mw.writeString("\n</tr>")
+	}
+	mw.writeString("</table>")
+}
+
+func (mw *mdWriter) infobox(n *types.InfoboxNode) {
+	mw.newBlock()
+	mw.writeString(`<aside markdown="1" class="`)
+	mw.writeString(string(n.Kind))
```

Comment:
Actually, you could just use `mw.writeFmt("<... class=%q>", n.Kind)`

But, because it's HTML now, you'll have to make sure it's safe using escaping. See this, for instance:
https://github.com/googlecodelabs/tools/blob/508f3fab2b82b08747277d730570444f1ab2c444/claat/render/html.go#L135